src/components/Rockets/Rockets.js

```diff
@@ -0,0 +1,46 @@
+import { useSelector, useDispatch } from 'react-redux';
+import './Rockets.css';
+import { bookRocket, cancelRocket } from '../../redux/Rockets/rockets';
+
+const Rockets = () => {
+  const rockets = useSelector((state) => state.rocketsReducer.rockets);
```

Comment:
- [ ] [OPTIONAL] since your reducers have a wait state. It would be nice if you rendered a loading display while the Missions, Rockets & Missions.